README.md

```diff
@@ -33,9 +33,14 @@ For more information about how to use RubyGems, see our RubyGems basics guide at
 
 ## Installation
 
-RubyGems is likely already installed in your Ruby environment, you can check by running `gem --version` in your terminal emulator.
-In some cases your OS's package manager may install RubyGems as a separate package from Ruby. It's recommended to check
-with your OS's package manager before installing RubyGems manually.
+RubyGems is already installed in your Ruby environment, you can check the version you have installed by running `gem --version` in your terminal emulator.
+
+In some cases Ruby & RubyGems may be provided as OS packages. This is not a
+recommended way to use Ruby & RubyGems. It's better to use a Ruby Version
+Manager, such as [rbenv](https://github.com/rbenv/rbenv). If you still want to
```

Comment:
Any particular reason for `rbenv`? The most simple ruby manager these days is IMHO `chruby`.

Reply:
Nah, just personal preference. Should I link to `chruby` as well? Technically both `rbenv` and `chruby` are version switchers, not installers, but I think they should both have good documentation on how to install ruby in a way that works with them.

Reply:
I was just curious. Definitely not blocker for me.

Reply:
I updated it!